Source/Core/VideoCommon/TextureConverterShaderGen.cpp

```diff
@@ -32,35 +32,49 @@ TCShaderUid GetShaderUid(EFBCopyFormat dst_format, bool is_depth_copy, bool is_i
 
 ShaderCode GenerateShader(APIType api_type, const UidData* uid_data)
 {
-  ShaderCode out;
+  const bool mono_depth = uid_data->is_depth_copy && g_ActiveConfig.bStereoEFBMonoDepth;
```

Comment:
No need for the const qualifier here.

Reply:
While it's not super important, using const is a bit helpful even for small things like this.